### ISSUE TITLE
Accessibility issue - color contrast for orange theme

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/ilw-content#11]
The orange theme is being flagged for insufficient color contrast on the content components:

- [ilw-content](https://builder3.toolkit.illinois.edu/preview/ilw-content/1-0/index.html)
- [ilw-content-inset](https://builder3.toolkit.illinois.edu/preview/ilw-content-inset/1-0/index.html)
- [ilw-content-introduction](https://builder3.toolkit.illinois.edu/preview/ilw-content-introduction/1-0/index.html)
- [ilw-content-lede](https://builder3.toolkit.illinois.edu/preview/ilw-content-lede/1-0/index.html)

The orange background and white text is giving a [contrast ratio of 3.04:1](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FF5F05), which is okay for large text. However, some text in the component is falling below the threshold for large text:

- 14 point (typically 18.66px) and bold or larger, or,
- 18 point (typically 24px) or larger.
